packages/codegen-ui-react/lib/__tests__/forms/validation.test.ts

```diff
@@ -158,75 +158,86 @@ describe('validateField tests', () => {
     ).toEqual({ hasError: true, errorMessage: 'test' });
   });
   it('should validate BE_AFTER type', () => {
-    const startDate = new Date().toDateString();
-    const endDate1 = new Date('2021-01-09').toDateString();
-    const endDate2 = new Date('3000-01-09').toDateString();
-    expect(
-      validateField(startDate, [{ type: ValidationTypes.BE_AFTER, strValues: [endDate1], validationMessage: '' }]),
-    ).toEqual({ hasError: false });
-    expect(
-      validateField(startDate, [{ type: ValidationTypes.BE_AFTER, strValues: [endDate2], validationMessage: '' }]),
-    ).toEqual({ hasError: true, errorMessage: `The value must be after ${endDate2}` });
-    expect(
-      validateField(startDate, [{ type: ValidationTypes.BE_AFTER, strValues: [''], validationMessage: 'test' }]),
-    ).toEqual({ hasError: true, errorMessage: 'test' });
-
+    // timestamp tests
     const startTime = Date.now();
-    const endTime1 = startTime - 10;
-    expect(
-      validateField(startTime, [
-        { type: ValidationTypes.BE_AFTER, strValues: [endTime1.toString()], validationMessage: '' },
-      ]),
-    ).toEqual({ hasError: false });
-    expect(
-      validateField(endTime1, [
-        { type: ValidationTypes.BE_AFTER, strValues: [startTime.toString()], validationMessage: '' },
-      ]),
-    ).toEqual({ hasError: true, errorMessage: `The value must be after ${startTime}` });
-    expect(
-      validateField(endTime1, [
-        { type: ValidationTypes.BE_AFTER, strValues: [startTime.toString()], validationMessage: 'test' },
-      ]),
-    ).toEqual({ hasError: true, errorMessage: 'test' });
-  });
-  it('should validate BE_BEFORE type', () => {
-    const startDate = new Date('2022-01-09').toString();
-    const endDate1 = new Date('2023-01-09').toString();
-    const endDate2 = new Date('2021-01-09').toString();
+    const endTime1 = startTime + 10;
+    const endTime2 = startTime - 10;
 
-    expect(
-      validateField(startDate, [
-        { type: ValidationTypes.BE_BEFORE, strValues: [endDate1.toString()], validationMessage: '' },
-      ]),
-    ).toEqual({ hasError: false });
-    expect(
-      validateField(startDate, [
-        { type: ValidationTypes.BE_BEFORE, strValues: [endDate2.toString()], validationMessage: '' },
-      ]),
-    ).toEqual({ hasError: true, errorMessage: `The value must be before ${endDate2}` });
-    expect(
-      validateField(startDate, [
-        { type: ValidationTypes.BE_BEFORE, strValues: [endDate2.toString()], validationMessage: 'test' },
-      ]),
-    ).toEqual({ hasError: true, errorMessage: 'test' });
+    const dateFormatTestCases = [
+      {
+        startDate: '01 Jan 2022',
+        endDate1: 'Jan 08 2023',
+        endDate2: '01 Jan 2021',
+      },
+      {
+        startDate: '01/09/2022',
+        endDate1: '01/09/2023',
+        endDate2: '01/09/2021',
+      },
+      {
+        startDate: '2022-01-09',
+        endDate1: '2023-01-09',
+        endDate2: '2021-01-09',
```

Comment:
I think we're missing an invalid date test

Reply:
Ill add it